ui/components/app/incoming-trasaction-toggle/incoming-transaction-toggle.tsx

```diff
@@ -88,7 +87,9 @@ const IncomingTransactionToggle = ({
         justifyContent={JustifyContent.spaceBetween}
```

Comment:
We could add a gap prop here so there is some space between the text and toggle button
```suggestion
        justifyContent={JustifyContent.spaceBetween}
        gap={4}
```

Reply:
Good catch ! will add to styles !

Reply:
@georgewrmarshall also added a storybook 🙏🏻 thanks for the review ❤️